content/04.discussion.md

```diff
@@ -16,34 +16,34 @@ We worked closely with pathologists and clinicians to build modules from which w
 
 We harnessed RNA expression data for a number of analyses, yielding important biological insights across multiple brain tumor histologies.
 For example, we performed subtyping of medulloblastoma tumors, for which only 35% (43/122) had subtype information from pathology reports. <!--SAMPLECOUNT-->
-Among the subtyped tumors, we accurately recapitulated subtypes using MM2S (91%; 39/43) or medulloPackage (95%; 41/43) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263]. <!--SAMPLECOUNT-->
-We then applied the consensus of these methods to subtype all medulloblastoma tumors lacking pathology-based subtypes.
+We accurately subtyped medulloblastomas with pathology-based subtypes using MM2S (91%; 39/43) or MedulloClassifier (95%; 41/43) [@doi:10.1186/s13029-016-0053-y; @doi:10.1371/journal.pcbi.1008263] and subsequently applied the consensus of these methods to subtype all medulloblastomas.
 
 We advanced the integrative analyses and cross-cohort comparison via a number of validated modules.
 We used an expression classifier to determine whether tumors have dysfunctional _TP53_ [@doi:10.1016/j.celrep.2018.03.076] and the EXTEND algorithm to determine their degree of telomerase activity using a 13-gene signature [@doi:10.1038/s41467-020-20474-9].
-Interestingly, in contrast to adult colorectal cancer and gastric adenocarcinoma, in which _TP53_ loss of function is less frequent in hypermutated tumors [@doi:10.18632/oncotarget.22783; @https://doi.org/10.1038/NATURE13480], we found that hypermutant HGG tumors universally displayed dysregulation of _TP53_.
+Interestingly, in contrast to adult colorectal cancer and gastric adenocarcinoma, in which _TP53_ loss of function is less frequent in hypermutated tumors [@doi:10.18632/oncotarget.22783; @https://doi.org/10.1038/NATURE13480], we found that hypermutant HGGs universally displayed dysregulation of _TP53_.
```

Comment:
This sentence confused me, so I rephrased to what I think it means. If it doesn't mean this, some other kind of rephrase should be done.
```suggestion
Interestingly, in contrast to adult colorectal cancer and gastric adenocarcinoma, in whose hypermutated tumors _TP53_ loss of function is less frequent [@doi:10.18632/oncotarget.22783; @https://doi.org/10.1038/NATURE13480], we found that hypermutant HGGs universally displayed dysregulation of _TP53_.
```